redirects.yaml

```diff
@@ -457,6 +457,7 @@ gateways/?:  "/internet-of-things/gateways"
 get/?: "/download"
 getubuntu.*/?: "/download"
 GetUbuntu.*/?: "/download"
+gov.*?: "/desktop/organisations"
```

Comment:
I think you mean just `gov/?`

Reply:
well, there was a contact-us and thank-you, so I wanted to get that too... do you see any problem with this? too greedy?  I did test on /community/goverance and it seemed ok.

Reply:
Ah of course, I wondered if this would pick up `/some/govern/docs` but I guess we don't have gov in other links.